pyaedt/edb_core/nets.py

```diff
@@ -292,9 +293,15 @@ def get_net_list(net_name, _net_list):
             new_ext = [all_nets[0]]
             get_net_list(new_ext[0], new_ext)
             all_nets = [i for i in all_nets if i not in new_ext]
-            self._extendend_nets.append(new_ext)
-
-        return self._extendend_nets
+            _extended_nets.append(new_ext)
+
+            if len(new_ext) > 1:
```

Comment:
I am not sure we need this if condition here.
Because if len(new_ext) == 1 it should work too.
But not if the len is ==0
```suggestion
            if len(new_ext) > 0:
```

Reply:
len(new_ext) > 1 is on purpose. only when there are more than one net, an extended net is created.